README.md

```diff
@@ -0,0 +1,79 @@
+# Conduit Connector Zendesk
+
+### General
+
+The Zendesk connector is one of [Conduit](https://github.com/ConduitIO/conduit) custom plugins. It provides both, a source
+and a destination zendesk connectors.
+
+### How to build it
+
+Run `make`.
+
+### Testing
+
+Run `make test` to run all the tests.
+
+## Zendesk Source
+
+The Zendesk client connector will connect with Zendesk API through the `url` constructed using subdomain specific to individual organization. Upon successful configuration with `zendesk.userName ` and `zendesk.apiToken` the tickets from the given domain is fetched using cursor based [incremental exports](https://developer.zendesk.com/api-reference/ticketing/ticket-management/incremental_exports/) provided by zendesk. The cursor is initiated with start_time set to `0` or the time set in `position` of last successfully ack'd record and all subsequent iterations are done using `next_url` returned by the zendesk till the pipeline is paused. On resuming of the pipeline updated_at time of the last fetched ticket is used to restart the cursor.
```

Comment:
> The cursor is initiated with start_time set to `0` or the time set in `position` of last successfully ack'd record and all subsequent iterations are done using `next_url` returned by the zendesk till the pipeline is paused. On resuming of the pipeline updated_at time of the last fetched ticket is used to restart the cursor.

So this means that tickets are always returned in ascending order, by updated_at?

Reply:
Yes, the tickets will order in ascending based on the updated_time.